pkg/webreg/webreg.go

```diff
@@ -471,6 +471,15 @@ image. The index image is named <code>ci-index</code> and can be exposed to test
 steps via the <a href="/help/ci-operator#literal-references"><code>dependencies</code></a> feature.
 </p>
 
+<p>
+Thi ephemeral index is built from scratch and only the bundles built in the
+current <code>ci-operator</code> run will be added to it, nothing else. The
+bundles are added to the index using the <code>semver</code> mode, which means
+that the CSV version needs to be a valid semver. Also, if the CSV has a <code>spec.replaces</code>
```

Comment:
nit: CSV stands for Cluster Service Version so "CSV Version" is like PIN Number. Maybe specifically call out `csv.spec.version`?